recipes/jsoncpp/all/conanfile.py

```diff
@@ -37,8 +37,8 @@ def _patch_sources(self):
 
         if self.settings.os != "Windows" and not self.options.shared and not self.options.fPIC:
             tools.replace_in_file(os.path.join(self._source_subfolder, "src", "lib_json", "CMakeLists.txt"),
-                                  "set_target_properties( jsoncpp_lib PROPERTIES POSITION_INDEPENDENT_CODE ON)",
-                                  "set_target_properties( jsoncpp_lib PROPERTIES POSITION_INDEPENDENT_CODE OFF)")
+                                  "POSITION_INDEPENDENT_CODE ON",
+                                  "POSITION_INDEPENDENT_CODE OFF")
```

Comment:
```suggestion
                                  "set_target_properties( jsoncpp_lib PROPERTIES POSITION_INDEPENDENT_CODE ON)",
                                  "")
```

Reply:
Unfortunately, this won't work with 1.9.3, because there is no such a lign.